### PR TITLE
Add Co-responsible

### PR DESCRIPTION
**Pull request by x183**
Added fields for a co-responsible in EditEvent

### CODE REVIEW

**Comment by x183** [comment]
not fixed =(

File: backend/src/services/party_report.service.ts [hunk 1]
```diff
@@ -4,35 +4,63 @@ import { to } from "../utils";
 import { statusChanged } from "./email_sender.service";
 
 export const validPartyReport = (party_report: PartyReport) => {
-  if (
-    !/^[\+]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,5}$/im.test(
-      party_report.responsible_number,
-    )
-  ) {
+  console.log(party_report);
```
Comment (molleer) [inline backend/src/services/party_report.service.ts:7]: Remove this

File: frontend/src/utils/utils.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
 import moment from "moment";
 
-export const formatDT = date => moment(Date(date)).format("yyyy-MM-DD HH:mm");
-export const formatDate = date => moment(Date(date)).format("yyyy-MM-DD");
-export const formatTime = date => moment(Date(date)).format("HH:mm");
+export const formatDT = date => {
+	console.log(date);
```
Comment (molleer) [inline frontend/src/utils/utils.js:4]: Remove this

**Review by molleer — CHANGES_REQUESTED** [review]
Looks nice, but the backend does not set co-responsible fields

**Pull request merged**